clients/admin-ui/src/features/user-management/UserForm.tsx

```diff
@@ -144,28 +166,50 @@ const UserForm = ({
               </Stack>
               <Divider mb={7} mt={7} />
               <Heading fontSize="xl" colorScheme="primary">
-                Privileges
+                Permissions
               </Heading>
-              <Text>Select privileges to assign to this user</Text>
-              <Divider mb={2} mt={2} />
+              <Text>Select permissions to assign to this user</Text>
+              <Box>
+                <Button
```

Comment:
I noticed the issue didn't really say what type of button to put here. will probably want to get this looked at by product to make sure they didn't want something more like a toggle or checkbox

Reply:
Good idea. @saiatethyca what do you think of the buttons used in this PR?

Reply:
Thanks @TheAndrewJackson this is great! couple of minor edits.
- Can we name Select all permissions to Select all, similarly Deselect all?
- Also, can we add some information below select/deselect buttons to instruct the user to click save again to apply the permissions?
  - Text can be "To apply the changes, click 'Save' below."

Reply:
Definitely, I'll make those edits now